FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_IP.c

```diff
@@ -2105,6 +2105,7 @@ uint8_t ucProtocol;
 		if( xResult != pdPASS )
 		{
 			FreeRTOS_printf( ( "xCheckSizeFields: location %ld\n", xLocation ) );
+			( void ) xLocation;
```

Comment:
I would add a comment here as to why this statement exists; same for line 2383.

Reply:
Done, thanks!